appscale/tools/remote_helper.py

```diff
@@ -889,12 +907,18 @@ def terminate_virtualized_cluster(cls, keyname, is_verbose):
       is_verbose: A bool that indicates if we should print the commands executed
         to stdout.
     """
-    AppScaleLogger.log("Terminating instances in a virtualized cluster with " +
-      "keyname {0}".format(keyname))
+    AppScaleLogger.log("Terminating appscale deployment with keyname" +
+      " keyname {0}".format(keyname))
     time.sleep(2)
 
     shadow_host = LocalState.get_host_with_role(keyname, 'shadow')
-    acc = AppControllerClient(shadow_host, LocalState.get_secret_key(keyname))
+    try:
+      acc = AppControllerClient(shadow_host, LocalState.get_secret_key(keyname))
```

Comment:
I would put the try/except around the LocalState call so that it's clear which call is throwing the exception.